database-mongodb/MySQL/initiateSeeding.js

```diff
@@ -1,20 +1,22 @@
 const mysql = require('mysql');
 const { mysqlHost, mysqlUser, mysqlPassword } = require('../../config.js');
+const data = require('../unsplashData.js');
 
 var connection = mysql.createConnection({
   host: mysqlHost,
   user: mysqlUser,
   password: mysqlPassword,
 });
 
-const table = 'CREATE TABLE IF NOT EXISTS SDC_Image_Service_MySQL.images (itemId VARCHAR(10) NOT NULL UNIQUE PRIMARY KEY, itemImages VARCHAR(90) NOT NULL)';
+const table = 'CREATE TABLE IF NOT EXISTS SDC_Image_Service_MySQL_10Million.images (itemId VARCHAR(10) NOT NULL UNIQUE PRIMARY KEY, itemImages VARCHAR(90) NOT NULL)';
+const badTable = 'CREATE TABLE IF NOT EXISTS SDC_Image_Service_MySQL_10Million.imagesBad (indexNumber INT NOT NULL UNIQUE PRIMARY KEY AUTO_INCREMENT, itemId VARCHAR(10) NOT NULL, itemImages VARCHAR(90) NOT NULL)';
```

Comment:
What about the badTable is bad necessarily? I see theres an indexNumber column in that table that doesn't exist in your original table. Is it just having the extra column?